src/main/java/uk/gov/companieshouse/overseasentitiesapi/validation/OverseasEntitySubmissionDtoValidator.java

```diff
@@ -160,10 +160,14 @@ private void validateTrustDetails(OverseasEntitySubmissionDto overseasEntitySubm
 
             if (!overseasEntitySubmissionDto.isForUpdateOrRemove()) {
                 // Note that this validation is only done for registrations
-                trustIndividualValidator.validate(overseasEntitySubmissionDto.getTrusts(), errors, loggingContext);
+                trustIndividualValidator.validate(overseasEntitySubmissionDto.getTrusts(), errors, loggingContext, overseasEntitySubmissionDto.isForUpdateOrRemove());
```

Comment:
I would hard-code a value of `false` for the final parameter here. Makes things clearer (its value must be `false` as processing has entered the block)

Reply:
done